docs/reference/deployment_options.md

```diff
@@ -32,6 +32,35 @@ Terraform module to deploy the 5G user plane in edge sites.
 
 `````
 
+## User Plane Function
+
+The User Plane Function (UPF) is available in two different charms:
+
+`````{tab-set}
+    
+````{tab-item} Kubernetes
+
+The [UPF Kubernetes Charm](https://charmhub.io/sdcore-upf-k8s) can be deployed on a Kubernetes cluster. Network Configuration is done using Multus. Unless the charm is deployed in DPDK mode, the charm will not modify the host network configuration. To deploy the Kubernetes charm, follow this [guide](/how-to/deploy_sdcore_cups/).
+
+````
+
+````{tab-item} Machine
+
+The [UPF Machine charm](https://charmhub.io/sdcore-upf) can be deployed on a bare metal machine or a VM. The UPF machine charm will modify the host network configuration:
+- Create network interfaces (DPDK mode only)
+- Set interfaces IP addresses
+- Modify interfaces MTU
+- Set MAC addresses on interfaces (DPDK mode only)
+- Bring interfaces up
+- Create IP routes
+- Create IP tables rules
+
+To deploy the Machine charm, follow this [guide](/how-to/deploy_sdcore_upf_machine/).
+
+````
```

Comment:
For the machine charm, maybe we should add the caveat that we do not have COS integration at this time. I think it is the biggest issue with that choice of deployment at this time.

Reply:
How come we don't have the COS integration? We have the `cos_agent` integration here.
https://github.com/canonical/sdcore-upf-operator/blob/main/charmcraft.yaml#L25

Reply:
I don't remember the exact details, but I think there are some issues with the logs in particular. I have never tested it myself. I'll add a ticket to Jira to validate this and approve this PR.